tests/ui-toml/min_rust_version/manual_non_exhaustive_msrv.rs

```diff
@@ -0,0 +1,19 @@
+mod enums {
```

Comment:
Is there a reason the test for this lint is split from the other lints?

Reply:
@flip1995 If I remember correctly, setting the `msrv` to the latest version in the test's toml was only resulting in error messages from this lint as it is evaluated in an early pass. Since we're testing with `1.0.0` I think this can be in the same file as other lints